Dockerfile

```diff
@@ -16,6 +16,6 @@ RUN pip install -U setuptools
 RUN pip install . --trusted-host github.com
 
 # Install gsutil to get crc32c and file size from cloud data
-RUN curl -sSL https://sdk.cloud.google.com | bash
+#$RUN curl -sSL https://sdk.cloud.google.com | bash
```

Comment:
do we want to comment this out? Maybe just remove it?

Reply:
Whoops yeah we can just revert this, it was something I was messing with